### PR TITLE
add localization for subscribe button

### PR DESCRIPTION
**Pull request by Arhell**
issue #20871

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
         	<input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="{{ T "input_placeholder_email_address" }}" aria-label="email" required>
             <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
             <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_3885586f8f1175194017967d6_11c1b8bcb2" tabindex="-1" value=""></div>
-            <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+            <div class="clear"><input type="submit" value="{{ T "subscribe_button" }}" name="{{ T "subscribe_button" }}" id="mc-embedded-subscribe" class="button"></div>
```
Comment (lex111) [inline layouts/index.html:31]: What is the point of localizing the `name` attribute?

Comment (Arhell) [reply layouts/index.html]: corrected

File: i18n/en.toml [hunk 2]
```diff
@@ -61,6 +61,9 @@ other = "Yes"
 [input_placeholder_email_address]
 other = "email address"
 
+[subscribe_button]
+other = "Subscribe"
+
```
Comment (sftim) [inline i18n/en.toml:66]: This file is sorted alphabetically, I think. Would it be OK to move this so the file stays sorted @Arhell?

Comment (Arhell) [reply i18n/en.toml]: done, thanks

**Pull request merged**